build-legato.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
Don't put shell script in top-level directory. Instead, always use GNU Make to pick up appropriate targets along with the shell scripts in classified directories.